CHANGELOG.md

```diff
@@ -3,3 +3,9 @@
 ## release-1.0.0
 
 Created initial repository.
+
+## release-1.1.0
```

Comment:
This naming might cause confusion, because you are not adding a new branch called "release-1.1.0"